### ISSUE TITLE
Compiler warning about variable length array 'emudata'

### ISSUE THREAD

**Issue by bieryAtFnal** [DUNE-DAQ/flxlibs#18]
This is a minor issue, but I noticed the following warning when compiling the flxlibs package with other DAQ software packages today.  

```
/home/biery/dunedaq/jcfTest/sourcecode/flxlibs/apps/emu_confgen.cxx: In function ‘int main(int, char**)’:
/home/biery/dunedaq/jcfTest/sourcecode/flxlibs/apps/emu_confgen.cxx:273:27: warning: ISO C++ forbids variable length array ‘emudata’ [-Wvla]
   uint64_t emudata[emusize];
                           ^
/home/biery/dunedaq/jcfTest/sourcecode/flxlibs/apps/emu_confgen.cxx:290:21: warning: comparison of integer expressions of different signedness: ‘int’ and ‘uint32_t’ {aka ‘unsigned int’} [-Wsign-compare]
   for (int i = 0; i < emusize; i++) {
```

The commit hash that I'm using in the flxlibs repo is 81e70cab2

**Comment by roland-sipos** [comment]
Thanks Kurt, indeed I need to lint this new app before making a tag candidate. Will fix it asap.

**Comment by roland-sipos** [comment]
Fixed in fe2d792
